src/routes/JobOpportunity/Overview.tsx

```diff
@@ -1,23 +1,29 @@
 /* eslint-disable sonarjs/no-duplicate-string */
-import { MainLayout, RoutesNavigationList, RoutesNavigationListProps, SimpleNavigationList, Title } from '@/components';
+import {
+  ActionButton,
+  FavoriteToggle,
+  LoginModal,
+  MainLayout,
+  RoutesNavigationList,
+  RoutesNavigationListProps,
+  SimpleNavigationList,
+  Title,
+} from '@/components';
 import { CompareCompetencesTable } from '@/components/CompareTable/CompareCompetencesTable';
 import { useToolStore } from '@/stores/useToolStore';
 import { getLocalizedText } from '@/utils';
-import { Accordion } from '@jod/design-system';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
+import { MdCompareArrows, MdOutlinePrint, MdOutlineRoute, MdOutlineShare } from 'react-icons/md';
 import { useOutletContext } from 'react-router-dom';
-import Tabs from './Tabs';
 import { LoaderData } from './loader';
 
 const Overview = () => {
```

Comment:
Remove the sub-route from the pathname and rename **Overview** to **JobOpportunity**.